paper-icon-button-light.html

```diff
@@ -45,6 +44,12 @@
         opacity: 0.6;
         color: currentColor;
       }
+      
+      img {
+        vertical-align: middle;
+        width: var(--iron-icon-width, 24px);
```

Comment:
@bicknellr I think I understand what you're saying, but given that an `iron-icon` inside the button would have a width of `--iron-icon-width`, making `img` the same would make it automatically consistent without users having to style it from the host.